src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -92,6 +103,7 @@ function AdminAddCoursesForm(props) {
     setIsModalVisible(false);
   };
 
+  const idToken = 'stub, remove and implement with state when auth0 is added';
```

Comment:
Let's remove this and not set false values for places where we'll want to implement `idToken`s!